src/lib/components/search/load/Loads.svelte

```diff
@@ -12,31 +14,45 @@
 
 	// parameters
 	export let query;
+	export let hidden = true;
+	export let delay = 100; // test value and change accordingly
 
 	// variables
 	const category = categoryFrom(query);
 	const numberOfResults = 32;
+
+	onMount(() => {
+		setTimeout(() => {
+			hidden = false;
+		}, delay);
+	});
 </script>
 
-{#if category === CategoryEnum.IMAGES}
-	<div class="px-4 py-8">
-		<section id="images" class="flex flex-wrap justify-center gap-2">
-			{#each { length: numberOfResults } as _}
-				<div class="flex-none">
-					<LoadImage />
-				</div>
-			{/each}
-		</section>
-	</div>
-{:else if category !== undefined}
-	<div class="sm:mx-auto mb-4 max-w-screen-sm">
-		<section id="result-list" class="mx-2 my-4 max-w-fit overflow-clip">
-			{#each { length: numberOfResults } as _}
-				<Load />
-				<hr class="my-2 border border-gray-200 dark:border-gray-600" />
-			{/each}
-		</section>
-	</div>
-{:else}
-	<Error statusCode={'500'} message={'Unknown category requested.'} />
-{/if}
+<div class:hidden>
```

Comment:
Fixed in https://github.com/hearchco/frontend/pull/124/commits/3dca96084b70f016c1ca54d62047572347d911fe

Please remove this :hidden class addition

Reply:
> IMO it would be better if we could have the animation for a fixed minimum delay instead of not showing anything for a delay.

Had this in mind, but I already committed so that's that. 3dca960 is a much better solution